src/services/leafletHelpers.js

```diff
@@ -100,7 +100,45 @@ angular.module("leaflet-directive").factory('leafletHelpers', function ($q, $log
         return !_isDefined(value);
     };
 
+    // BEGIN DIRECT PORT FROM AngularJS code base
```

Comment:
I ended up not using this; so I should possibly remove. But going down the `lf-` path later we possibly might need this.

Let me know if I should kill this.